### ISSUE TITLE
Try testing Image Builder images in TF directly

### ISSUE THREAD

**Issue by TomasTomecek** [packit/packit-service#1973]
* [ ] share with `125523088429`
* [ ] in `us-east-2`
* [ ] pass AMI name to Testing Farm (which parameter?)
* [ ] record a demo

**Comment by thrix** [comment]
Opened a tracking issue on our side:  https://issues.redhat.com/browse/TFT-1981

In case any work would be needed

**Comment by lachmanfrantisek** [comment]
Hi @thrix, here is an image that should be shared with your account:
* `us-east-1`:  https://us-east-1.console.aws.amazon.com/ec2/home?region=us-east-1#ImageDetails:imageId=ami-05c538dfb064a4ec0
* `us-east-2`: https://us-east-2.console.aws.amazon.com/ec2/home?region=us-east-2#ImageDetails:imageId=ami-0f539832672d9c272

It comes from here: https://github.com/packit/hello-world/pull/1363/checks?check_run_id=12666355704

I hope it's correctly shared with your account.

**Comment by lachmanfrantisek** [comment]
@thrix can you please take a look? Looks like the previous one isn't shared with your account but this one should:

```
ami-0a435713562680fde
```

**Comment by thrix** [comment]
yeah, seems with this quartely delay I have ... tried to kick it myself

**Comment by mfocko** [comment]
@lachmanfrantisek to discuss with @thrix about the future of this card
